src/cdk/overlay/dispatchers/overlay-keyboard-dispatcher.ts

```diff
@@ -28,7 +32,11 @@ export class OverlayKeyboardDispatcher extends BaseOverlayDispatcher {
 
     // Lazily start dispatcher once first overlay is added
     if (!this._isAttached) {
-      this._document.body.addEventListener('keydown', this._keydownListener);
+      this._ngZone
+        ? this._ngZone.runOutsideAngular(() =>
```

Comment:
Can you use an `if/else` statement here, rather than the ternary? Also can you leave an `@breaking-change` note here as well? That way our tooling will pick it up when we're doing the breaking changes.

Reply:
Yep, done.